03/app.js

```diff
@@ -0,0 +1,11 @@
+const btnsList = document.querySelectorAll('button')
+const button = document.querySelector('button')
+
+const buttonClicked = function(e) {
+    console.log('Clicked')
+    this.innerText = 'clicked'
```

Comment:
Zadanie mówi o wykonaniu tylko raz "kliknięcia". Obecnie można klikać wielokrotnie na każdy button i widzimy efekt tego w konsoli - trzeba to poprawić :)